### DISCUSSION TITLE
Cursed Binds

### DISCUSSION THREAD

**Discussion by alicealysia** [YaLTeR/niri#750] [Show and tell]
:smiley: 

   Mod+Alt+1 { set-column-width "25%"; }
    Mod+Alt+2 { set-column-width "33.33333%"; }
    Mod+Alt+3 { set-column-width "50%"; }
    Mod+Alt+4 { set-column-width "66.66666%"; }
    Mod+Alt+5 { set-column-width "75%"; }
    Mod+KP_End { set-column-width "25%"; }
    Mod+KP_Down { set-column-width "33.33333%"; }
    Mod+KP_Next { set-column-width "50%"; }
    Mod+KP_Left { set-column-width "66.66666%"; }
    Mod+KP_Begin { set-column-width "75%"; }

    Mod+Alt+Alt+KP_Home { spawn "bash" "-c" "niri msg action set-column-width '33.33333%' && niri msg action move-column-to-first && niri msg action consume-window-into-column";}
    Mod+Alt+KP_Left { spawn "bash" "-c" "niri msg action expel-window-from-column && niri msg action set-column-width '33.33333%' && niri msg action move-column-to-first";}
    Mod+Alt+KP_End { spawn "bash" "-c" "niri msg action move-column-to-first && niri msg action consume-or-expel-window-right && niri msg action set-column-width '33.33333%'";}
    Mod+Alt+KP_UP { spawn "bash" "-c" "niri msg action consume-window-into-column && niri msg action set-column-width '33.33333%'";}
    Mod+Alt+KP_Begin { spawn "bash" "-c" "niri msg action expel-window-from-column && niri msg action set-column-width '33.33333%' && niri msg action center-column";}
    Mod+Alt+KP_DOWN { spawn "bash" "-c" "niri msg action consume-or-expel-window-left && niri msg action set-column-width '33.33333%'";}   
    Mod+Alt+KP_Prior { spawn "bash" "-c" "niri msg action move-column-to-last && niri msg action consume-or-expel-window-left && niri msg action move-window-up && niri msg action set-column-width '33.33333%'";}
    Mod+Alt+KP_RIGHT { spawn "bash" "-c" "niri msg action expel-window-from-column && niri msg action set-column-width '33.33333%' && niri msg action move-column-to-last";}
    Mod+Alt+KP_Next { spawn "bash" "-c" "niri msg action move-column-to-last && niri msg action consume-or-expel-window-left && niri msg action set-column-width '33.33333%'";}

    Mod+ctrl+KP_Home { spawn "bash" "-c" "niri msg action set-column-width '25%' && niri msg action move-column-to-first && niri msg action consume-window-into-column";}
    Mod+ctrl+KP_Left { spawn "bash" "-c" "niri msg action expel-window-from-column && niri msg action set-column-width '25%' && niri msg action move-column-to-first";}
    Mod+ctrl+KP_End { spawn "bash" "-c" "niri msg action move-column-to-first && niri msg action consume-or-expel-window-right && niri msg action set-column-width '25%'";}
    Mod+ctrl+KP_UP { spawn "bash" "-c" "niric msg action consume-window-into-column && niri msg action set-column-width '50%'";}
    Mod+ctrl+KP_Begin { spawn "bash" "-c" "niri msg action expel-window-from-column && niri msg action set-column-width '50%' && niri msg action center-column";}
    Mod+ctrl+KP_DOWN { spawn "bash" "-c" "niri msg action consume-or-expel-window-left && niri msg action set-column-width '50%'";}   
    Mod+ctrl+KP_Prior { spawn "bash" "-c" "niri msg action move-column-to-last && niri msg action consume-or-expel-window-left && niri msg action move-window-up && niri msg action set-column-width '25%'";}
    Mod+ctrl+KP_RIGHT { spawn "bash" "-c" "niri msg action expel-window-from-column && niri msg action set-column-width '25%' && niri msg action move-column-to-last";}
    Mod+ctrl+KP_Next { spawn "bash" "-c" "niri msg action move-column-to-last && niri msg action consume-or-expel-window-left && niri msg action set-column-width '25%'";}
`

**Comment by YaLTeR** [comment]
Obviously, Mod+numpad should instead rotate the entire layout in 3D corresponding to Blender camera views

**Reply by alicealysia** [reply]
you're so right, what was I thinking???

**Reply by alicealysia** [reply]
Huh? Would I really learn GLSL for a meme?
Probably not... but what if it was for wobbly windows... hmmmmmm
